### PR TITLE
docs: simplify component descriptions

### LINKED ISSUE #293: [Docs]  Shorten component descriptions for better discovery
**What is the goal of the documentation**
Let's cut the component descriptions a bit so that they are more visually pleasing.

### PR DESCRIPTION
**Pull request by stijnmoreels**
Simplify the component descriptions to a single, short sentence.

Closes #293

### CODE REVIEW

File: website/resources/components/azure-ml.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Azure ML
-description: With Arcus we are offering an open source library that streamlines Azure ML development, but lets ML engineers focus on the actual job at hand, without losing time in tinkering with the AzureML SDK and all overhead that comes with it.
+description: Streamline Azure ML development, but lets ML engineers focus on the actual job at hand, without losing time in tinkering with the AzureML SDK.
```
Comment (pim-simons) [inline website/resources/components/azure-ml.md:3]:
```suggestion
description: Streamline Azure ML development and lets ML engineers focus on the actual job at hand, without losing time in tinkering with the AzureML SDK.
```

maybe this is a little better?

File: website/resources/components/background-jobs.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Background Jobs
-description: Arcus BackgroundJobs provides common repeated background tasks that strengthen an application. This goes from automatic secret invalidation of Azure Key Vault secrets to publish an event on Azure EventGrid upon a potential expired client secret in Azure. This collection of background tasks has grown from real-life application builders that wanted to re-use existing functionality in a safe and trusted manner.
+description: Strengthen you application with commonly repeated tasks ranging from automatic secret invalidation of Azure Key Vault secrets to publish an event on Azure EventGrid and more.
```
Comment (pim-simons) [inline website/resources/components/background-jobs.md:3]:
```suggestion
description: Strengthen your application with commonly repeated tasks ranging from automatic secret invalidation of Azure Key Vault secrets to publish an event on Azure EventGrid and more.
```

**Review by pim-simons — APPROVED** [review]
Looks good to me, some small remarks 👍🏻

File: website/resources/components/azure-ml.md (hunk 1 above)
Comment (THMKAE) [reply website/resources/components/azure-ml.md:3]:
```suggestion
description: Streamline Azure ML development and let ML engineers focus on the actual job at hand, without losing time in tinkering with the AzureML SDK.
```
What about this?

File: website/resources/components/background-jobs.md (hunk 2 above)
Comment (THMKAE) [reply website/resources/components/background-jobs.md:3]:
```suggestion
description: Strengthen your application with commonly repeated tasks ranging from automatic secret invalidation of Azure Key Vault secrets to publishing events on Azure EventGrid and more.
```

File: website/resources/components/templates.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Templates
-description: Arcus templates provides several .NET project templates which already includes all the best practices and boilerplate code to kickstart your project. We provide several types of templates to accommodate the most popular types of projects.
+description: Kickstart your project with .NET project templates which already includes all the best practices and boilerplate code.
```
Comment (THMKAE) [inline website/resources/components/templates.md:3]:
```suggestion
description: Kickstart your project with .NET project templates which already include all the best practices and boilerplate code.
```

File: website/resources/components/web-api.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Web API
-description: The Arcus Web API library provides an easy way to register and configure complex functionality to your application. We provide ways to add shared access key authentication, HTTP correlation, request tracking, strict JSON formatting, and other boilerplate code that can be added with zero effort. 
+description: Simplified API development with shared access key authentication, HTTP correlation, request tracking, strict JSON formatting, and other boilerplate code that can be added with zero effort.
```
Comment (THMKAE) [inline website/resources/components/web-api.md:3]:
```suggestion
description: Simplify your API development with shared access key authentication, HTTP correlation, request tracking, strict JSON formatting, and other boilerplate code.
```

File: website/resources/components/observability.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Observability
-description: Arcus Observability allows you to work easily with Azure Application Insights telemetry by making use of the common ILogger infrastructure to track dependencies, log custom metrics and log multi-dimensional telemetry data.
+description: Track Azure Application Insights dependencies, log custom metrics and log multi-dimensional telemetry data via the common ILogger infrastructure.
```
Comment (THMKAE) [inline website/resources/components/observability.md:3]:
```suggestion
description: Track Azure Application Insights dependencies, log custom metrics, and log multi-dimensional telemetry data via the common ILogger infrastructure.
```

File: website/resources/components/ml.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus ML
-description: With Arcus we are offering an open source library that streamlines ML development with Python and offers out of the box implementations for common scenarios.
+description: Streamline ML development with Python and bring out of the box implementations for common scenarios.
```
Comment (THMKAE) [inline website/resources/components/ml.md:3]:
```suggestion
description: Streamline ML development with Python and bring out-of-the-box implementations for common scenarios.
```

File: website/resources/components/messaging.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Messaging
-description: Arcus Messaging is a library that helps with the integration of messaging systems like queues or event subscriptions and allows you to focus on implementing business logic when processing messages.
+description: Focus on implementing business logic when processing messages from queues or event subscription.
```
Comment (THMKAE) [inline website/resources/components/messaging.md:3]:
```suggestion
description: Focus on implementing business logic when processing messages from queues or event subscriptions.
```

File: website/resources/components/observability.md (hunk 5 above)
Comment (stijnmoreels) [reply website/resources/components/observability.md:3]: Ah, yes, the English way of listing. 😄

**Review by THMKAE — APPROVED** [review]

**Review by fgheysels — APPROVED** [review]
Nice & sharp!

**Pull request merged**